detection-rules/impersonation_wells_fargo.yml

```diff
@@ -9,7 +9,7 @@ severity: "high"
 source: |
   type.inbound
   and (
-      strings.ilike(sender.display_name, '*wells fargo*')
+      sender.display_name =~ 'wells fargo'
```

Comment:
no substrings anymore? sounds fine to me, otherwise you may want `strings.icontains`

Reply:
yeah no substring anymore, too many FPs